public/index.html

```diff
@@ -13,6 +13,10 @@
       content="Coding Bootcamp Europe-#Class2: Inga Schallock, Christian Glissmann, Christian Lux, Deniz Dalsar, Simon Brings, Atef Amri"
     />
     <meta name="google" content="notranslate" />
+    <script
```

Comment:
Font Awesome ist ja in der Regel recht groß - sind das alle icons oder kann man das konfigurieren?

Reply:
"It because FontAwsome is a font eventually, so just like you can't use only the letters Q, n and L of a specific font, you can't use only some of the icons of that font "

Gibt tooling um das zu umgehen, aber ein sehr schlauer Lehrer meinte mal zu mir "Nur Tooling verwenden, was man wirklich braucht" :D